packages/zowe-explorer/src/trees/uss/USSAttributeView.ts

```diff
@@ -45,14 +43,13 @@ export class USSAttributeView extends WebView {
         switch (message.command) {
             case "refresh":
                 if (this.canUpdate) {
-                    this.onUpdateDisposable = this.ussNode.onUpdate(async (node) => {
-                        await this.attachTag(node);
-                        await this.panel.webview.postMessage({
-                            attributes: await this.ussNode.getAttributes(),
-                            name: node.fullPath,
-                            readonly: this.ussApi.updateAttributes == null,
-                        });
-                        this.onUpdateDisposable.dispose();
+                    const attrs = await this.ussNode.fetchAttributes();
```

Comment:
It feels a little scary not to check if the function `fetchAttributes` exists since it's optional.

Reply:
oh, I didn't see this as a comment in OG PR.  I agree and can add it here then it can port back into main with 3.1.2 updates, what do you think @zFernand0 & @traeok ?

Reply:
we can merge if this makes you feel more comfortable @zFernand0

Reply:
This helps a bunch! 🙏